docs/getting-started/cash-app-pay.md

```diff
@@ -0,0 +1,213 @@
+---
+layout: default
+title: Cash App Pay
+parent: Getting Started
+nav_order: 4
+---
+
+# Cash App Pay
+{: .d-inline-block .no_toc }
+NEW (v4.3.0)
+{: .label .label-green }
+
+
+<details markdown="block">
+  <summary>
+    Table of contents
+  </summary>
+  {: .text-delta }
+- TOC
+{:toc}
+</details>
+
+
+{: .alert }
+> Cash App Pay is currently available in the following region(s): US
+
+Cash App Pay support can be added using your Afterpay merchant account. This requires a token to be generated by a server-to-server call to the Afterpay API Create Checkout endpoint with a parameter of `isCashAppPay` set to `true`. This method requires importing and implementing the Cash App Pay Kit SDK.
+
+{: .info }
+> When creating a checkout token, both `redirectConfirmUrl` and `redirectCancelUrl` must be set. Failing to do so will cause undefined behavior. The SDK’s example merchant server sets the parameters [here][example-server-props]{:target='_blank'}. See more by checking the [api reference][api-reference-props]{:target='_blank'}.
+
+## Step 1: Import the Cash App Pay Kit Dependency
+
+You can get the the latest version of the SDK from Maven. This is the import definition using Gradle:
+
+```gradle
+implementation "app.cash.paykit:core:1.0.3"
+```
+
+For definitions of other build systems, see [Cash App Pay Kit on Maven Central][cash-on-maven]{:target="_blank"}.
+
+{: .info }
+> Version `v1.0.3` of the SDK size is `12.3 kB`.
+
+## Step 2: Create a Cash App Pay Kit SDK Instance
+
+To create a new instance of the Cash App Pay Kit SDK, you must pass the `clientId`. This is a required field. This can be retrieved through the Afterpay object: `Afterpay.environment.payKitClientId`.
+
+{: .note }
+> Ensure that the Afterpay SDK is configured per the [instructions][configure-afterpay] before attempting to access `Afterpay.environment.payKitClientId`
```

Comment:
@ScottAntonacAP the link to instructions is broken, please check

Reply:
Thanks, the link is only broken in the html that I passed across because it was a saved file that didn't include the whole site. The link works on my local with the dev server running and will work once live. Really appreciate the thorough testing 👍